README.md

```diff
@@ -500,6 +500,10 @@ using the name of the corresponding environment variable.
   applications to work with the downside of not cleanly terminating clvk. Use
   with caution.
 
+* `CLVK_DEVICE_EXTENSIONS` specifies extensions to be added to the list of
+  exposed extensions by `clGetDeviceInfo` in `CL_DEVICE_EXTENSIONS`. It
```

Comment:
```suggestion
  exposed extensions. It
```
This now no longer reflects what the change does. Maybe keeping it abstract is easier?